test/public/runs/runsPerLhcPeriod.overview.test.js

```diff
@@ -178,24 +181,15 @@ module.exports = () => {
     });
 
     it('can navigate to a run detail page', async () => {
-        await reloadPage(page);
-        await page.waitForSelector('tbody tr');
-
-        const expectedRunNumber = await page.evaluate(() => document.querySelector('tbody tr:first-of-type a').innerText);
-
-        await waitForNavigation(page, async () => await pressElement(page, 'tbody tr:first-of-type a'));
-        const redirectedUrl = await page.url();
-
-        const urlParameters = redirectedUrl.slice(redirectedUrl.indexOf('?') + 1).split('&');
-
-        expect(urlParameters).to.contain('page=run-detail');
-        expect(urlParameters).to.contain(`runNumber=${expectedRunNumber}`);
+        const expectedRunNumber = await getInnerText(await page.waitForSelector('tbody tr:first-of-type a'));
+        await waitForNavigation(page, () => pressElement(page, 'tbody tr:first-of-type a'));
+        expectUrlParams(page, { page: 'run-detail', runNumber: expectedRunNumber });
     });
 
     const EXPORT_RUNS_TRIGGER_SELECTOR = '#export-runs-trigger';
 
     it('should successfully export all runs per lhc Period', async () => {
-        await goToPage(page, 'runs-per-lhc-period', { queryParameters: { lhcPeriodName: 'LHC22a' } });
+        await waitForNavigation(page, () => page.goBack());
```

Comment:
You are directly adding dependency on the test before here, you should avoid this.

Reply:
applied